docker_push_latest_if_changed.py

```diff
@@ -1,5 +1,120 @@
 #!/usr/bin/env python3
 import argparse
+import hashlib
+from collections import namedtuple
+
+import docker_util
+from verbose_print import print_v
+from verbose_print import print_vv
+from verbose_print import set_print_level
+
+
+ImageParityFields = namedtuple(
+    'ImageParityFields',
+    ('image_cmds_hash', 'packages_hash'),
+)
+
+GET_PACKAGE_LIST_COMMAND = ['dpkg', '-l']
+
+
+def _docker_push_latest_if_changed(source, target, is_dry_run):
```

Comment:
I usually find it best to hide boolean traps behind keyword-only arguments so it's _forced_ that they're document.

for here:

```python
def _docker_push_latest_if_changed(source, target, *, dry_run):
```

also probably no reason to prefix it with `is_`  `if dry_run` reads just fine

Reply:
Sounds good. I enjoy that behavior as well
Probably going to keep the `is_` prefix for the sake of convention though